packages/fuel-indexer-api-server/src/uses.rs

```diff
@@ -392,6 +387,21 @@ pub async fn run_query(
     }
 }
 
+pub async fn gql_playground(
+    Path((namespace, identifier)): Path<(String, String)>,
+) -> impl IntoResponse {
+    let html = playground_source(
+        GraphQLPlaygroundConfig::new(&format!("/api/graph/{}/{}", namespace, identifier))
+            .with_setting("scehma.polling.enable", false),
+    );
+
+    Response::builder()
+        .status(StatusCode::OK)
+        .header(http::header::CONTENT_TYPE, "text/html; charset=utf-8")
+        .body(Body::from(html))
+        .expect("Failed to build gql playground response.")
```

Comment:
Convention: unwrap with `?`
```suggestion
        ?
```

Reply:
I had to implement the `From` method on enums `ApiError` and `HttpError` in order to satisfy the compiler. This allows us to use ? here.